### ISSUE TITLE
pretrained retinanet model

### ISSUE THREAD

**Issue by myopengit** [tensorflow/models#9982]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [ ] I checked to make sure that this feature has not been requested already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/official/vision/detection/main.py

## 2. Describe the feature you request

The default retinanet 50 model achieves 37.4 map in the pytorch Detectron 2 Model zoo. What is your accuracy of your retinanet model? Can you publish a pretrained model?

**Comment by izeitak** [comment]
On a similar issue, can you provide the config file used to train? There is no link in:
https://github.com/tensorflow/models/blob/master/official/vision/beta/MODEL_GARDEN.md

**Comment by saberkun** [comment]
Hi folks, we are a bit behind because we need legal approval for artifact releases.
@xianzhidu I think we are getting legal approval on releasing the artifacts, right?

**Comment by jvishnuvardhan** [comment]
@myopengit @izeitak Are you looking for the following config
https://github.com/tensorflow/models/blob/master/official/vision/configs/retinanet.py
https://github.com/tensorflow/models/blob/master/official/vision/MODEL_GARDEN.md

**Comment by izeitak** [comment]
This is what I see for Apache 2.0:

[cid:ace3c59a-4b09-48fb-815e-c325881eba0a]
________________________________

**Comment by itai12312** [comment]
<img width="575" alt="image" src="https://user-images.githubusercontent.com/5047231/162134938-2b398d20-6733-4bc1-bba6-bef11bcea212.png">
I still don't see it in the model_garden.md

**Comment by itai12312** [comment]
Hi, is there a response from the legal team?
